plugins/builddep.py

```diff
@@ -62,9 +63,32 @@ def __exit__(self, exc_type, exc, exc_tb):
 class BuildDepCommand(dnf.cli.Command):
 
     aliases = ('builddep',)
-    msg = "Install build dependencies for .src.rpm, .nosrc.rpm or .spec file"
+    msg = "Install build dependencies for .src.rpm, .nosrc.rpm, .spec file or repo"
     summary = _(msg)
-    usage = _("[PACKAGE.src.rpm|PACKAGE.nosrc.rpm|PACKAGE.spec]")
+    usage = _("[PACKAGE.src.rpm|PACKAGE.nosrc.rpm|PACKAGE.spec|PACKAGE]")
```

Comment:
Yes, see the reference to Panu's comment in https://bugzilla.redhat.com/show_bug.cgi?id=1074585.